src/main/java/com/mixpanel/android/viewcrawler/EditState.java

```diff
@@ -159,7 +171,12 @@ public void run() {
             }
             // ELSE View is alive and we are alive
 
-            mEdit.visit(viewRoot);
+            try {
+                mEdit.visit(viewRoot);
+            } catch (ViewVisitor.LayoutUpdateException e) {
```

Comment:
This seems like it may be to early to catch the error- we'll continue visiting every time the system updates the global view layout, and could throw in there, too.